layers/auto-completion/README.org

```diff
@@ -180,6 +180,7 @@ Here is an example to add =company= auto-completion to python buffer:
 | ~C-j~       | go down in company dropdown menu                                         |
 | ~C-k~       | go up in company dropdown menu                                           |
 | ~C-/~       | search in company dropdown                                               |
```

Comment:
I mean the documentation only adds a line, but does not replace the stale info for `C-/` (there are two rows)